pycbc/workflow/core.py

```diff
@@ -324,13 +324,9 @@ def add_ini_opts(self, cp, sec):
 
                 # Get LFN and PFN
                 for path in values:
-                    # Here I decide if the path is URL or
-                    # IFO:/path/to/file or IFO:url://path/to/file
-                    # That's somewhat tricksy as we used : as delimiter
-                    split_path = path.split(':', 1)
+                    split_path = path.split(':',1)
```

Comment:
Why did you remove my comment? Also why remove the space that now makes this not PEP8 compliant.

Reply:
This is just from when you sent me this code through slack, I never updated it to when you fixed it properly. Fixing this now.